src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/TransferringBodyRepository.scala

```diff
@@ -16,4 +16,9 @@ class TransferringBodyRepository(db: Database) {
 
     db.run(query.result).map(body => body.head)
   }
+
+  def dbHasTransferringBodies(implicit executionContext: ExecutionContext): Future[Boolean] = {
+    val query = Body.length
```

Comment:
I know the other method in here has an execution context on the method. Now there's two methods with it, could you put it on the class level and remove it from the method level.